Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public HttpWebRequest httpWebRequest { get; set; }
+        public HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper(string contentType = "application/json")
+        {
+            httpWebRequest.ContentType = contentType;
+        }
+        
+        public string Get(string url, Dictionary<string,string> parameters)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (!(parameters is null) || parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+            httpWebRequest.Method = "GET";
+            HttpWebRequest.Create(url);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream, Encoding.UTF8);
+            var result = streamReader.ReadToEnd();
+            return result;
+        }
+
+        public string Put(string url, Dictionary<string, string> parameters, string body)
+        {
+            if (string.IsNullOrEmpty(url))
```

Comment:
why we dont thorw exception here, but throw exception in getUrlWithParams?

Reply:
Here I prefer not to ruin all program to have the opportunity to work without breaking everything. 
But I am not sure who will use private function . For example, we could not check message and error will go deeper

Reply:
Are we sure that everyone who call this method made try - catch ?